src/views/chainState/index.tsx

```diff
@@ -51,6 +54,8 @@ const ChainState = () => {
   const [storageSelected, setStorageSelected] = useState(palletSelected?.storage?.items?.at?.(0));
 
   const [callArgs, setCallArgs] = useState<unknown>(undefined);
+  const [encodedStorageKey, setEncodedStorageKey] = useState<string | undefined>('0x');
```

Comment:
string | undefined this typing looks a bit strange. Is there particular reason we want to set it to undefined?

Reply:
changed to <string>